Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -3571,6 +3573,7 @@ void HTMLMediaElement::seekTask()
 
     // 11 - Set the current playback position to the given new playback position
     m_seekRequested = true;
+    m_player->aboutToSeek(MediaTime::invalidTime());
     m_player->seekToTarget({ time, negativeTolerance, positiveTolerance });
```

Comment:
Why not have `seekToTarget` clear it automatically?

Reply:
because seekToTarget has an implementation in all the MediaPlayerPrivate ;

Reply:
Then do it in MediaPlayer?

Reply:
The MediaSourcePrivate only has access to the MediaPlayerPrivate

Reply:
```
void MediaPlayer::seekToTarget(const SeekTarget& target)
{
    m_private->aboutToSeek(MediaTime::invalidTime());
    m_private->seekToTarget(target);
}
```